tgui/packages/tgui/interfaces/Skyrat_SolGovGTFO.js

```diff
@@ -0,0 +1,61 @@
+import { toTitleCase } from 'common/string';
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, Section, Table, NoticeBox} from '../components';
+import { Window } from '../layouts';
+
+export const Skyrat_SolGovGTFO = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    lift_status,
+    list_of_riders,
+  } = data;
+  return (
+    <Window
+      width={315}
+      height={440}>
+      <NoticeBox danger>
+        Authorized SolGov personnel only
+      </NoticeBox>
+      <Window.Content>
+        <Section title="SolGov Fastpass™ Lift">
+          <LabeledList>
+            <LabeledList.Item label="Status">
+              {lift_status}
+            </LabeledList.Item>
+            <LabeledList.Item
+              buttons={(
+                <Button
+                  content="Activate Lift"
+                  onClick={() => act('activate_lift')} />
+              )}>
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        <Section title="Current riders">
+          <Table>
+            <Table.Row header>
+              <Table.Cell>
+                Rider
+              </Table.Cell>
+              <Table.Cell collapsing textAlign="right">
+                Occupation
+              </Table.Cell>
+            </Table.Row>
+            {list_of_riders.map(found_rider => {
+              <Table.Row>
+                <Table.Cell>
+                  {toTitleCase(found_rider.get_authentification_name())}
+                </Table.Cell>
+                <Table.Cell collapsing textAlign="right">
+                  <Box color="label" inline>
+                    {found_rider.get_assignment()}
+                    </Box>
+                </Table.Cell>
+              </Table.Row>
+            })};
```

Comment:
```suggestion
            {list_of_riders.map(found_rider => (
              <Table.Row>
                <Table.Cell>
                  {toTitleCase(found_rider.get_authentification_name())}
                </Table.Cell>
                <Table.Cell collapsing textAlign="right">
                  <Box color="label" inline>
                    {found_rider.get_assignment()}
                    </Box>
                </Table.Cell>
              </Table.Row>
            ))}
```
I could be wrong about this particular suggestion but I'm pretty sure this is the correct way to do it.

Reply:
eh, the whole thing needs to be looked back over, the list in its entirety dont work. I'm gonna commit these suggestions though so they fix the formatting a tad bit